spec/peeps_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'Peeps'
+require 'database_helpers'
+
+describe Peeps do
+  describe '.all' do
+    it 'returns all chitter posts' do
+      connection = PG.connect(dbname: 'chitter_test')
+      time_when_tested = "2022-07-15 12:00:15"
+      peep = Peeps.create(peep: "Hello this is my unit test for .all", time: time_when_tested)
+      Peeps.create(peep: "Hello this another peep from my unit test", time: time_when_tested)
+      peeps = Peeps.all
```

Comment:
You're missing a line that asserts something about the value of `peeps`. Currently this test doesn't test (assert) anything.